docs/icicle/colab-instructions.md

```diff
@@ -0,0 +1,131 @@
+# Run Icicle on Google Colab
+
+Google Colab let's you use a GPU free of charge, it's a Nvidia T4 GPU with 16 GB of memory, capable of running latest Cuda (tested on Cuda 12.2)
+As Colab is able to interact with shell commands, a user can also install a framework and load git repositories into Colab space.
+
+## Prepare Colab environment
+
+First thing to do in a notebook is to set the runtime type to a T4 GPU.
+* in the upper corner click on the dropdown menu and select "change runtime type"
+
+<p align="center">
```

Comment:
U add images like this

![Change runtime](../../static/img/colab_change_runtime.png)